### PR TITLE
OKTA-527004 : Error link focus fix 

### PR DESCRIPTION
**Pull request by leemchale-okta**
## Description:

This PR fixes the issue of the `Back to sign in` link being not in focus when the session expired error appears.

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-527004](https://oktainc.atlassian.net/browse/OKTA-527004)

### CODE REVIEW

File: src/v3/src/transformer/layout/email/emailChallengeTransformer.ts [hunk 1]
```diff
@@ -53,6 +53,9 @@ export const transformEmailChallenge: IdxStepTransformer = ({ transaction, formB
     'credentials.passcode',
     uischema.elements as UISchemaElement[],
   );
+  if (passcodeElement) {
+    passcodeElement.focus = true;
+  }
```
Comment (glenfannin-okta) [inline src/v3/src/transformer/layout/email/emailChallengeTransformer.ts:58]: Why was this needed? Shouldn't it be found as the first element in the transformer?

**Review by glenfannin-okta — COMMENTED** [review]
Can you also add a test to [this](https://github.com/okta/okta-signin-widget/blob/0.0/src/v3/test/integration/error-session-expired.test.tsx#L18) spec and just asserting that the back to sign in link is focused.

File: src/v3/src/transformer/layout/email/emailChallengeTransformer.ts (hunk 1 above)
Comment (leemchale-okta) [reply src/v3/src/transformer/layout/email/emailChallengeTransformer.ts:58]: this was needed because the Enter a code button flow and the Enter code input flow are combined in the same transformer in a stepper layout.  So only the Enter code button is focused because it is the first element.  So after you press the Enter a code button in order for the Enter code input to be focused, I had to manually add focus to that input component.
